app/assets/stylesheets/home.css.scss

```diff
@@ -1,7 +1,7 @@
 
 
 /*===Fonts===*/
-@import url(http://fonts.googleapis.com/css?family=Roboto+Slab);
+@import url(//fonts.googleapis.com/css?family=Roboto+Slab);
```

Comment:
Got rid of the http because we can't do unsecure http requests when we are connecting with https. Cool tip: if you just type "//" in front of the URL, the browser automagically senses whether to do an http request or an https request based on the type of connection you have.